selfdrive/controls/lib/alertmanager.py

```diff
@@ -19,7 +19,7 @@ def set_offroad_alert(alert: str, show_alert: bool, extra_text: Optional[str] =
     a = OFFROAD_ALERTS[alert]
     if extra_text is not None:
       a = copy.copy(OFFROAD_ALERTS[alert])
-      a['text'] += extra_text
+      a['extra'] = extra_text
```

Comment:
It works now, but do you think it would be better to always set extra to avoid some possible key errors in the future?

Reply:
y,that should be better